### ISSUE TITLE
Access gateway auto-upgrades are failing due to unsatisfied dependency on python3-six

### ISSUE THREAD

**Issue by xjtian** [magma/magma#791]
Debian-based autoupgrade to latest 1.0.0-rc1 tag fails on production gateways:

```
magma@magma:~$ sudo apt-get install --dry-run magma=1.0.0-1571929094-1837b3be
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 magma : Depends: python3-six (>= 1.12.0) but 1.10.0-3 is to be installed
E: Unable to correct problems, you have held broken packages.
```

**Comment by xjtian** [comment]
Resolved by 4c30efb10ada3a8f5b9c12c244184a866e53689d

**Comment by xjtian** [comment]
Verified fixed on release branch
